lib/example/example.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Plexy.Example do
+  use Plug.Router
+  plug Plexy.RequestId
+  plug Plexy.Instrumentor
```

Comment:
Also not exactly sure how to wrap all the Plexy stuff into one thing so it just becomes `plug Plexy`